prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,19 @@
+let languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+
+console.log(languages);
+
+var firstElement = languages[0];
+
+console.log("The first element of the array is " + firstElement);
+
+let thirdElement = languages[2];
+
+console.log("The 3rd element of the array is " + thirdElement);
+
+let length = languages.length;
+
+console.log("The length of the array is " + length);
+
+let lastElement = languages[5];
```

Comment:
While this will work it will break if the size of the array changes. We want to programmatically get the last element in the array by using the `length` of the array minus `1`